learn/react-native/flashlight.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Flashlight"
```

Comment:
Change title for better SEO. Write what flashlight is used for

Reply:
Updated